shared/services/config/smartnode-config.go

```diff
@@ -572,6 +587,24 @@ func NewSmartnodeConfig(cfg *RocketPoolConfig) *SmartnodeConfig {
 			config.Network_Holesky: {},
 		},
 
+		previousRocketNetworkPricesAddresses: map[config.Network][]common.Address{
+			config.Network_Mainnet: {
+				common.HexToAddress("0x751826b107672360b764327631cC5764515fFC37"),
+			},
+			config.Network_Prater:  {},
+			config.Network_Devnet:  {},
+			config.Network_Holesky: {},
```

Comment:
We're gonna need to put the Holesky addresses in here and below since eventually that'll get upgraded to Houston and price / balance reporting will stop working with v1.12 on that while Atlas is still live.